packages/web/src/scss/components/Navigation/README.md

```diff
@@ -0,0 +1,82 @@
+# Navigation
+
+The `Navigation` component is a container for the navigation links of the application.
+
+It consists of a these parts:
+
+- [Navigation](#navigation)
+- [NavigationLink](#navigationlink)
+
+## Navigation
+
+The `Navigation` is a `nav` wrapper for lists of links or other navigation components.
+
+```html
+<nav class="Navigation">
+  <ul>
+    <li>
+      <!-- link -->
+    </li>
+  </ul>
+</nav>
+```
+
+It centers its children vertically and if the children are not NavigationLink components,
+it will apply a gap between them.
+
+## NavigationLink
+
+The `NavigationLink` is component that is styled to be used as a navigation link.
```

Comment:
Isn't it obvious? 🤔

Reply:
It is, but I like to have an introduction sentence into the component :))